2.4-micro/s2i/bin/assemble

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
```

Comment:
`2.4-micro/s2i/` directory looks exactly the same as in `2.4/s2i`, `2.4-micro/s2i/` could  be just a symlink to `2.4/s2i/`.

Reply:
I think we do not use symlinks for `s2i` directories, they are real copies, but may be I am wrong.

Reply:
That was my point, now they are copies, but we can remove duplication by having symlinks.

Reply:
Please create a symlink as @hhorak proposes. It makes sense to me as well.